suite-common/wallet-core/src/blockchain/blockchainActions.ts

```diff
@@ -0,0 +1,56 @@
+import { createAction } from '@reduxjs/toolkit';
+
+import { NetworkSymbol } from '@suite-common/wallet-config';
+import type { CustomBackend, NetworksFees } from '@suite-common/wallet-types';
+import type { Timeout } from '@trezor/type-utils';
+
+export const actionsPrefix = '@common/blockchain';
+
+const connected = createAction(`${actionsPrefix}/CONNECTED`, (payload: NetworkSymbol) => ({
+    payload,
+}));
+
+type ReconnectTimeoutStartPayload = {
+    symbol: NetworkSymbol;
+    id: Timeout;
+    time: number;
+    count: number;
+};
+const reconnectTimeoutStart = createAction(
+    `${actionsPrefix}/RECONNECT_TIMEOUT_START`,
```

Comment:
Since we give names for all our newly created actions in camel case, I would prefer it here too => `reconnectTimeoutStart`.

Same for `${actionsPrefix}/CONNECTED` in this file.